FlowCrypt/Core/CoreHost.swift

```diff
@@ -147,3 +149,29 @@ extension SecPadding {
     // https://developer.apple.com/documentation/security/secpadding/ksecpaddingnone
     public static let NONE = SecPadding(rawValue: 0)
 }
+
+
+private func modPow<T: BinaryInteger>(n: T, e: T, m: T) -> T {
```

Comment:
Where is this from?

Reply:
https://rosettacode.org/wiki/Modular_exponentiation#Swift
Will also add a comment there

Reply:
I'm very hesitant to use a code grabbed from a wiki for modular exponentiation. It does look correct, but I'm not a cryptographer. There's a chance of side channel leaks, like timing leaks. On the other hand, in the context of our app, that should not be a problem.

Given that this is a beta, I'll merge it. We'll have to have a cryptographer have a deeper look at this before we move to prod.

Reply:
Should I leave some extra comments there to not to forget this before moving to prod?

Reply:
For sure. Plus please make an issue, and add the issue link in the code comment. Also, please move it into a separate file. Thanks!

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/238